docs/pages/application-access/guides/connecting-apps.mdx

```diff
@@ -77,9 +79,7 @@ proxy_service:
   title="Web Proxy Port"
 >
   Teleport uses [TLS-ALPN-01](https://letsencrypt.org/docs/challenge-types/#tls-alpn-01)
-  ACME challenge to validate certificate requests which only works on port `443`.
-  As such, in order to use ACME for certificate management, web proxy needs to
-  be accessible on port `443`.
+  ACME challenge to validate certificate requests which only works on port `443`.  Make sure your Teleport proxy is accessible on port `443` when using ACME for certificate management.
```

Comment:
I've noticed we ACME is coming up in our top searches. 
![image](https://user-images.githubusercontent.com/559288/120830685-9d5b7800-c513-11eb-917c-c58c01498120.png)

Maybe we can add a dedicated section to highlight and overview this feature.

Reply:
This might be useful in the planned Configuration article going into the Setup Section.